WhaleSpotting/Repositories/WhaleSightingRepo.cs

```diff
@@ -32,4 +34,20 @@ public WhaleSighting GetById(int id)
             throw new ArgumentOutOfRangeException($"No sightning with id {id} found in the database", ex);
         }
     }
+
+    public List<WhaleSightingResponse> ListApprovedSightings()
+    {
+        try
+        {
+            return context.WhaleSightings.Where(ws => (int)ws.ApprovalStatus == 1)
+            .Include(ws => ws.User)
+            .Include(ws => ws.WhaleSpecies)
+            .Select(x => new WhaleSightingResponse(x))
+            .AsEnumerable().ToList();
```

Comment:
Can you do this without the `.AsEnumerable()` statement? Or does it give an error?
```suggestion
            .ToList();
```

Reply:
We get a 404 error if we take away .AsEnumerable(), so it's left in.